build-tools/packages/build-cli/src/commands/generate/typetests.ts

```diff
@@ -33,11 +33,11 @@ export default class GenerateTypetestsCommand extends PackageCommand<
 	static readonly description = "Generates type tests for a package or group of packages.";
 
 	static readonly flags = {
-		level: Flags.string({
+		level: Flags.custom<ApiLevel>({
 			description: "What API level to generate tests for.",
 			default: ApiLevel.internal,
 			options: knownApiLevels,
-		}),
+		})(),
```

Comment:
FYI, it seems that you can place `()` next to `custom<>` as well. Seems like it would keep things similar / easier to see the special extra need.